payment-intent/curl/payment-intent-auto.sh

```diff
@@ -0,0 +1,8 @@
+curl -v --request POST \
+    --url https://api.dojo.tech/payment-intents \
+    --header 'content-type: application/json' \
+    --header 'Authorization: Basic sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ' \
+    --header 'version: 2022-05-26' \
```

Comment:
2022-04-07 (it will be easier to update the version later if it is the same date)